app/gui/view/graph-editor/src/shortcuts.rs

```diff
@@ -21,10 +21,10 @@ pub const SHORTCUTS: &[(ensogl::application::shortcut::ActionType, &str, &str, &
     (Press, "!read_only", "cmd g", "collapse_selected_nodes"),
     // === Visualization ===
     (Press, "!node_editing", "space", "press_visualization_visibility"),
-    (DoublePress, "!node_editing", "space", "double_press_visualization_visibility"),
+    (Press, "!node_editing", "shift space", "open_fullscreen_visualization"),
```

Comment:
Should this condition be `!node_editing & !is_fs_visualization_displayed` to avoid sending a spurious `open_fullscreen_visualization` event along with the `close_fullscreen_visualization` event?

Reply:
Good call